CMakeLists.txt

```diff
@@ -466,16 +469,7 @@ else()
     nuttx PRIVATE $<GENEX_EVAL:$<TARGET_PROPERTY:nuttx,NUTTX_COMPILE_OPTIONS>>)
 endif()
 
-# Compiler options TODO: move elsewhere
-
-if("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU")
-  if(CMAKE_C_COMPILER_VERSION VERSION_GREATER 4.9)
-    # force color for gcc > 4.9
-    add_compile_options(-fdiagnostics-color=always)
-  endif()
-endif()
-
-if(MSVC)
```

Comment:
ditto

Reply:
this part has moved to ./arch/arm/src/cmake/gcc.cmake